pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.builds

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
That's a legitimate concern. I wasn't sure what the right name should be and this is only relevant for running UWP apps using CoreCLR so I started with this as a first effort. Perhaps Microsoft.NETCore.CoreCLRBuildTools? Do you have any suggestions?